src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -167,7 +167,7 @@ public static String wrap(String in, int wrapAmount) {
 
         String[] lines = in.split("\n");
         StringBuilder result = new StringBuilder();
-        addWrappedLine(result, lines[0], wrapAmount);
+        addWrappedLine(result, lines[0].replaceAll("\\s+$", ""), wrapAmount);
```

Comment:
Regarding the wrapping: I think, the entry point for the discussion is https://github.com/JabRef/jabref/issues/114. Think we agreed not not wrap single-line fields and that multiline fields should not be merged into one line.